crates/fraud_proofs/global_merkle_root/storage/src/update.rs

```diff
@@ -528,6 +557,50 @@ mod tests {
         assert!(coin_doesnt_exist_after_process_input);
     }
 
+    #[test]
+    /// When encountering a blob transaction,
+    /// `process_transaction` should insert the
+    /// corresponding blob.
+    fn process_transaction__should_insert_blob() {
+        let mut rng = StdRng::seed_from_u64(1337);
+
+        // Given
+        let blob = vec![1, 3, 3, 7];
+        let blob_id = BlobId::compute(&blob);
```

Comment:
what happens when we already have a blob that already exists? should we throw an error or just overwrite it?